server/remote_cache/cachetools/cachetools.go

```diff
@@ -66,13 +67,20 @@ func ComputeDigest(in io.ReadSeeker, instanceName string) (*digest.InstanceNameD
 	return digest.NewInstanceNameDigest(d, instanceName), nil
 }
 
-func ComputeFileDigest(fullFilePath, instanceName string) (*digest.InstanceNameDigest, error) {
-	f, err := os.Open(fullFilePath)
+func ComputeFileDigest(fullFilePath, instanceName string) (d *digest.InstanceNameDigest, err error) {
+	f, err := os.Open(filepath.Clean(fullFilePath))
```

Comment:
Does gosec treat non-clean file paths as a security issue? Asking because in most cases it will be a no-op since we usually generate file paths that don't include `../` etc., so it feels a bit like unnecessary boilerplate.

Reply:
It does. I can add `#nosec` directives instead, but I try to do that as little as possible with static analysis tools, since I feel like a lot of the point of them is to prevent future problems. If you're worried about the performance hit, though, that's what we'd do.

Reply:
That's a good point about future problems. I'm not too worried about a performance hit -- I guess reading the file is probably a few orders of magnitude more expensive than the `filepath.Clean`